### PR TITLE
HTML CSS prestations

### PR DESCRIPTION
**Pull request by Erdnaxela45**

### CODE REVIEW

**Comment by khadimb** [comment]
revue ok

**Review by khadimb — APPROVED** [review]
revue ok

File: public/assets/css/style.css [hunk 1]
```diff
@@ -5,4 +5,103 @@
...
+  
+.heading {
+      padding: 20px;
+      font-size: 20px;
+}
+  
+.card {
+      width: 80%;
+}
 }
```
Comment (LucyMerlier) [inline public/assets/css/style.css:107]: this may cause conflicts with other PRs. Also avoid magic values

File: src/View/Home/index.html.twig [hunk 2]
```diff
@@ -1,5 +1,50 @@
...
+            </div>
+          </div>
+        </div>
+        <div class="prestations-footer">
+            <p>Pour la voyance à distance, la consultation pourra être effectée une fois le paiement terminé via Western Union ou par chèque.</p>
+            <p>Compte tenu de la situation sanitaire, je ne peux vous recevoir au cabinet que si vous habitez proche de Chécy.</p>
+            <p>Sachez également qu'il n'y a aucune distinction quant à la qualité de la voyance au cabinet et à distance.</p>
+        </div>
+    </div>
+</section>
```
Comment (LucyMerlier) [inline src/View/Home/index.html.twig:49]: incoherent indentation? sometimes 2 spaces sometimes 4?

File: src/View/Home/index.html.twig [hunk 3]
```diff
@@ -1,5 +1,50 @@
...
+                <div class="card-body">
+                    <p>Tarif :</p>
+                    <p>75€ (pour 1 heure)</p>
+                    <p>50€ (pour 30 minutes)</p>
+                    <p>Consultation par téléphone ou visioconférence</p>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-header">
+                    <h3>Formation</h3>
```
Comment (LucyMerlier) [inline src/View/Home/index.html.twig:34]: there is a "card-title" class in bootstrap ;)

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments, also please be coherent when naming branches, and name them correctly

**Pull request closed without merging**